src/vue-wrapper.ts

```diff
@@ -62,10 +62,10 @@ export class VueWrapper<T extends ComponentPublicInstance> {
     return true
   }
 
-  emitted(): Record<string, unknown[]> {
+  emitted(eventName?: string): Record<string, unknown[]> | undefined {
```

Comment:
```suggestion
  emitted(): Record<string, unknown[]> | undefined
  emitted<T = unknown>(eventName: string): T[] | undefined
  emitted(
    eventName?: string
  ): Record<string, unknown[]> | unknown[] | undefined {
```

Reply:
@dobromir-hristov when you get a chance can you add this and make sure it's compiling? And we can merge :) no rush!

Reply:
Ok I did it 👍